packages/react-components/src/components/table-output-component.tsx

```diff
@@ -617,9 +617,16 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                     if (this.timestampCol) {
                         this.startTimestamp = this.endTimestamp = BigInt(rowNode.data[this.timestampCol]);
                     }
-                    this.handleRowSelectionChange();
-                    rowNode.setSelected(true);
+                    let itemPropsObj;
+                    // Notify selection changed
+                    this.handleRowSelectionChange(itemPropsObj);
```

Comment:
yes, true. The order matters, because the row selection signal might have selected a timegraph element and which would override the events table selection in the tooltip window. I think selections in the events table should be shown in the tooltip window because it was actively done in the table.